cmd/provenanced/cmd/init.go

```diff
@@ -9,13 +9,14 @@ import (
 	"strings"
 	"time"
 
-	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 
 	tmos "github.com/tendermint/tendermint/libs/os"
 	tmrand "github.com/tendermint/tendermint/libs/rand"
 	"github.com/tendermint/tendermint/types"
 
+	errors "cosmossdk.io/errors"
```

Comment:
I'm surprised `importas` doesn't complain about this. It's imported as `cerrs` everywhere else in the provenance repo. It's also needlessly aliased to `errors` here.